src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -1697,9 +1697,8 @@ def test_sql_vnet_mgmt(self, resource_group, resource_group_location, server):
 
         # Vnet 1
         self.cmd('network vnet create -g {} -n {}'.format(rg, vnetName1))
-        self.cmd('network vnet subnet create -g {} --vnet-name {} -n {} --address-prefix {} --service-endpoints {}'
```

Comment:
Maybe comment that you don't provide `--service-endpoints` so that later `-i` is useful? (I am just guessing how this feature works, sorry if I misunderstand)